core/modules/wdb/QueryAction.cc

```diff
@@ -284,7 +284,9 @@ void QueryAction::Impl::_transmit(bool last) {
     std::string resultString;
     _result->set_continues(!last);
     if (!_multiError.empty()) {
-        std::string msg = _multiError.toString();
+
+        std::string chunkId = std::to_string((*_msg).chunkid());
```

Comment:
This does not compile with gcc44. Please try to build on NCSA cluster before sending code for review (always).
You can use `boost::lexical_cast` before we all switch to more decent gcc version